ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -415,7 +425,16 @@ export class AccountCreation extends React.Component<
     return fp.isEmpty(errors) ? undefined : errors;
   }
 
+  // We need to stop showing demographic survey only after Nov-03
+  isInternational = () => {
```

Comment:
this name is only partially correct

Reply:
updated the name, however we will be removing the date check after 11/03